charts/pega/values-large.yaml

```
@@ -17,6 +17,12 @@ global:
   actions:
     execute: "deploy"
 
+  # New versions of the Pega Platform require that certain libraries are included in the bootstrap classpath
```

"New" will become outdated eventually - let's call out the specific version.
```suggestion
  # Pega Platform versions 25.0 and above require that certain libraries are included in the bootstrap classpath
```